[2023-05-05 07:25:03,041] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.extract_from_s3 manual__2023-05-05T07:24:48.889079+00:00 [queued]>
[2023-05-05 07:25:03,111] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.extract_from_s3 manual__2023-05-05T07:24:48.889079+00:00 [queued]>
[2023-05-05 07:25:03,113] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-05-05 07:25:03,114] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2023-05-05 07:25:03,114] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-05-05 07:25:03,204] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): extract_from_s3> on 2023-05-05 07:24:48.889079+00:00
[2023-05-05 07:25:03,214] {standard_task_runner.py:52} INFO - Started process 646 to run task
[2023-05-05 07:25:03,239] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_etl', 'extract_from_s3', 'manual__2023-05-05T07:24:48.889079+00:00', '--job-id', '1044', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_taskflow.py', '--cfg-path', '/tmp/tmplu3cqcua', '--error-file', '/tmp/tmp89vg1ndu']
[2023-05-05 07:25:03,250] {standard_task_runner.py:80} INFO - Job 1044: Subtask extract_from_s3
[2023-05-05 07:25:03,700] {task_command.py:370} INFO - Running <TaskInstance: dag_etl.extract_from_s3 manual__2023-05-05T07:24:48.889079+00:00 [running]> on host 295d97741f96
[2023-05-05 07:25:04,417] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Victor
AIRFLOW_CTX_DAG_ID=dag_etl
AIRFLOW_CTX_TASK_ID=extract_from_s3
AIRFLOW_CTX_EXECUTION_DATE=2023-05-05T07:24:48.889079+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-05T07:24:48.889079+00:00
[2023-05-05 07:25:04,426] {s3.py:861} INFO - Downloading source S3 file from Bucket raw with path supermarket_sales.csv
[2023-05-05 07:25:04,552] {base.py:68} INFO - Using connection ID 'minio_conn' for task execution.
[2023-05-05 07:25:04,557] {base_aws.py:206} INFO - Credentials retrieved from login
[2023-05-05 07:25:05,556] {python.py:173} INFO - Done. Returned value was: /opt/***/data/***_tmp_69cely59
[2023-05-05 07:25:05,949] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_etl, task_id=extract_from_s3, execution_date=20230505T072448, start_date=20230505T072503, end_date=20230505T072505
[2023-05-05 07:25:06,066] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-05 07:25:06,458] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
